python/huggingfaceserver/huggingfaceserver/model.py

```diff
@@ -251,7 +252,7 @@ async def predict(self, input_batch: Union[BatchEncoding, InferRequest], context
             try:
                 with torch.no_grad():
                     if self.task == MLTask.text2text_generation.value or self.task == MLTask.text_generation:
-                        outputs = self.model.generate(**input_batch)
+                        outputs = self.model.generate(**input_batch, pad_token_id=self.tokenizer.pad_token_id)
```

Comment:
is there a test case we can add ?

Reply:
Added

Reply:
the test seems to be failing
https://github.com/kserve/kserve/actions/runs/8417844462/job/23047080962?pr=3535

Reply:
fixed